dataland-backend/src/main/kotlin/org/dataland/datalandbackend/entities/DataMetaInformationEntity.kt

```diff
@@ -64,7 +64,11 @@ data class DataMetaInformationEntity(
     fun isDatasetViewableByUser(viewingUser: DatalandAuthentication?): Boolean {
         return this.qaStatus == QAStatus.Accepted ||
             this.uploaderUserId == viewingUser?.userId ||
-            viewingUser?.roles?.contains(DatalandRealmRole.ROLE_ADMIN) ?: false
+            viewingUser?.roles?.contains(DatalandRealmRole.ROLE_ADMIN) ?: false ||
+            (
+                viewingUser?.roles?.contains(DatalandRealmRole.ROLE_REVIEWER) ?: false &&
+                    this.qaStatus == QAStatus.Pending
+                )
```

Comment:
Was ist eigentlich mit datasets, die rejected wurden?  Sollten die nicht viewable sein von Reviewern?
Könnte relevant sein in Fällen, wo man nochmal "nachprüfen" möchte, ob man etwas zurecht rejected hat.

Reply:
Da wir die aber über den QA endpunkt für ungereviewte datasets aber nicht zurückgeben, kann der reviewer da nicht viel mit anfangen. Ich würde das vorerst so lassen. solange unser QA Prozess eher minimalistic ist.

Reply:
Backlog item erstellt dafür.  Ich close es daher.